mobile-app/app/api/transaction/dfi_converter.ts

```diff
@@ -42,12 +42,15 @@ export function dfiConversionCrafter (amount: BigNumber, mode: ConversionMode, o
   return {
     sign: async (account: WhaleWalletAccount) => await dfiConversionSigner(account, amount, mode),
     title: translate('screens/ConvertConfirmScreen', 'Converting DFI'),
+    displayTitle: translate('screens/ConvertConfirmScreen', 'Convert {{amount}} DFI to {{target}}', {
```

Comment:
Can we replace`title` instead of introducing another props?

Reply:
done, removed `displayTitle` and use `title` instead